src/localkiwixserver.cpp

```diff
@@ -24,16 +25,30 @@ LocalKiwixServer::LocalKiwixServer(QWidget *parent) :
 
     connect(ui->KiwixServerButton, SIGNAL(clicked()), this, SLOT(runOrStopServer()));
     connect(ui->OpenInBrowserButton, SIGNAL(clicked()), this, SLOT(openInBrowser()));
-    connect(KiwixApp::instance()->getSettingsManager(), &SettingsManager::portChanged,
-            this, [=](int port) { m_port = port; });
     connect(ui->closeButton, &QPushButton::clicked, this, &LocalKiwixServer::close);
-    const QHostAddress &localhost = QHostAddress(QHostAddress::LocalHost);
-    for (const QHostAddress &address: QNetworkInterface::allAddresses()) {
-        if (address.protocol() == QAbstractSocket::IPv4Protocol && address != localhost) {
-            m_ipAddress = address.toString();
-            break;
+    connect(ui->PortChooser, &QLineEdit::textChanged, ui->PortChooser, [=](const QString &text){
+        if(text.toInt() > 65535 || text.toInt() < 1) {
```

Comment:
Please put a space after the `if`

Reply:
Done along with some inconsistency in ipAddress related identifiers